vitrivr-engine-core/src/main/kotlin/org/vitrivr/engine/core/features/AbstractBatchedExtractor.kt

```diff
@@ -37,13 +67,6 @@ abstract class AbstractBatchedExtractor<C : ContentElement<*>, D : Descriptor<*>
     /** The [KLogger] instance used by this [AbstractExtractor]. */
     protected val logger: KLogger = KotlinLogging.logger {}
 
-    /** The names of the content source to consider during processing. */
-    protected val contentSources : Set<String>?
```

Comment:
Why can we remove this?

Reply:
Because it's now part of its superclass. Since every extractor appears to have it - something I architecturally disagree with, but that's a topic for a different discussion - it makes sense to have it defined only once as high up in the class hierarchy as possible.

Reply:
Ah yes, I didn't check the superclass. Makes sense.